### ISSUE TITLE
[React Native Gallery -> Permissions]: After invoking ‘Request’ button and navigating with tab key, focus is falling on invisible text.

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#312]
**Repro-Steps:**
Launch the React Native Gallery app.
Navigate to ‘Permissions’ button and invoke it.
Navigate to ‘Request’ button and invoke it.
After invoking ‘Request’ button navigate with tab key and observe.

**Actual Result:**
After invoking ‘Request’ button and navigating with tab key, focus is falling on invisible text.

**Expected Result:**
Focus should not fall on the invisible text and static text when user navigate with tab key. And all the text should be clearly visible to the user.

**User impact:**
Users with motor impairment who rely on keyboard will be confused if keyboard focus is falling on the invisible text.  

Internal bug link: https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/43361578
